.github/workflows/projects.yml

```diff
@@ -0,0 +1,19 @@
+name: Add new issues and pull requests to project board
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v1.0.2
+        with:
+          project-url: https://github.com/orgs/open-atmos/projects/6
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
@AgnieszkaZaba, thank you!
Do I understand correctly that for this to work, we need to setup a new secret at https://github.com/open-atmos/PySDM/settings/secrets/actions ?

Reply:
Yes.

Information from [Add to project action](https://github.com/actions/add-to-project)
> github-token (required) is a [personal access token](https://github.com/settings/tokens/new) with repo and project scopes. See [Creating a PAT and adding it to your repository](https://github.com/open-atmos/PySDM/pull/1446#creating-a-pat-and-adding-it-to-your-repository) for more details